chat-plugins/economy.js

```diff
@@ -815,7 +832,9 @@ exports.commands = {
 				Wisp.lastSeen(userid, online => {
 					Wisp.getTitle(userid, title => {
 						Wisp.getBackground(userid, background => {
-							showProfile(bucks, regdate, online, title, background);
+							Wisp.getMusic(userid, song => {
+								showProfile(bucks, regdate, online, title, background, song);
+							});
 						});
 					});
 				});
```

Comment:
@panpawn imo just create a function that passes an array/object containing all of the user's info to the callback function, instead of nesting all those async functions